Doc/about.rst

```diff
@@ -1,10 +1,10 @@
-=====================
-About these documents
-=====================
+========================
+About this documentation
+========================
 
 
-These documents are generated from `reStructuredText`_ sources by `Sphinx`_, a
-document processor specifically written for the Python documentation.
+These documents are generated from `reStructuredText`_ sources using `Sphinx`_,
+a documentation processor originally created for Python.
```

Comment:
Though not a native speaker, I'm fine with this current wording. It's also correct to say that it's a document processor *specifically* (I like that term explicitly) created for the Python documentation (and not for Python overall). Historically, we used to write the docs in LaTeX and moved to reST afterwards.

Reply:
```
This documentation is generated from `reStructuredText`_ sources using `Sphinx`_,
a document processor originally created for Python.
```
How about this?

Reply:
This looks good and properly convenes the intent that Sphinx was for Python. Let's ask my fellow Sphinx maintainer (and native English speaker) @AA-Turner for his opinion as well (we'll anyway need a core dev for merging).